site/content/docs/5.2/components/button-group.md

```diff
@@ -40,6 +40,13 @@ These classes can also be added to groups of links, as an alternative to the [`.
 
 ## Mixed styles
 
+<!-- Boosted mod : design callout -->
+{{< ods-incompatibility-alert >}}
+This variant should not be used because it does not respect the Orange Design System specifications.
+
+Please refer to the [Buttons: standard](https://system.design.orange.com/0c1af118d/p/278ebc-buttons-standard/b/539dce) guidelines on the Orange Design System website.
```

Comment:
The reference does not really explain why because we can see there green, red and orange buttons.